translations/monero-core_pt-br.ts

```diff
@@ -1102,27 +1102,27 @@
     <message>
         <location filename="../pages/Settings.qml" line="525"/>
         <source>Wallet seed &amp; keys</source>
-        <translation type="unfinished"></translation>
+        <translation>Keys da semente</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="527"/>
         <source>Secret view key</source>
-        <translation type="unfinished"></translation>
+        <translation>View key secreta</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="528"/>
         <source>Public view key</source>
-        <translation type="unfinished"></translation>
+        <translation>View key pública</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="529"/>
         <source>Secret spend key</source>
-        <translation type="unfinished"></translation>
+        <translation>Spend key secreta</translation>
```

Comment:
Thanks for explaining.
Better to leave as "View key" and "Spend key" unless there is a crystal clear translation which would improve user's comprehension. Users will google the terms if they don't know the difference between key types and find answers they need that way.